src/VMware/examples/Get-AzVMwarePlacementPolicy.md

```diff
@@ -0,0 +1,22 @@
+### Example 1: List
```

Comment:
Please provide a semantic title for example

Reply:
Done